code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -56,7 +56,14 @@
 	..()
 	reagents.add_reagent("synaptizine", 30)
 
+obj/item/weapon/reagent_containers/hypospray/combat/adminordrazine
+	desc = "A modified air-needle autoinjector for use in combat situations. Prefilled with adminordrazine for rapid healing."
+	volume = 100
 
+/obj/item/weapon/reagent_containers/hypospray/combat/adminordrazine/New()
+	..()
+	reagents.remove_reagent("synaptizine", 30)
+	reagents.add_reagent("adminordrazine", 100)

```

Comment:
Adminordrazane is ridiculously powerful, ten units is probably enough to
heal anybody to full health and give them a lot of stun immunity. Imo it
should be a lot of tricord or DD instead.
Op 7-jan.-2015 05:16 schreef "Steelpoint" <notifications@github.com>:

> In code/modules/reagents/reagent_containers/hypospray.dm
> <https://github.com/tgstation/-tg-station/pull/6705#discussion-diff-22570413>
> :
>
> >
> > +/obj/item/weapon/reagent_containers/hypospray/combat/adminordrazine/New()
> > +	..()
> > +	reagents.remove_reagent("synaptizine", 30)
> > +	reagents.add_reagent("adminordrazine", 100)
>
> That's 10 use's on 10 people. If we are sending in the ERT we can assume
> the station is utterly screwed and medbay may well be gone.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/6705/files#r22570413>.
>